openmeetings-web/src/main/webapp/css/raw-tree.css

```diff
@@ -17,6 +17,12 @@
 }
 .file-tree .footer {
 	height: var(--tree-footer-height);
+	display: flex;
+    flex-direction: column;
```

Comment:
whitespaces are inconsistent here (the whole code indented with tabs)

Reply:
my tab size in eclipse was wrong yeah

Reply:
you've got tab size 2 whitespaces I assume ?

Reply:
I have option "Show whitespaces" turned ON :))